src/reducers/zipped-profiles.js

```diff
@@ -33,6 +33,23 @@ function zipFile(
   action: Action
 ): ZipFileState {
   switch (action.type) {
+    case 'UPDATE_URL_STATE': {
+      if (
+        action.newUrlState.pathInZipFile === null &&
+        state.phase === 'VIEW_PROFILE_IN_ZIP_FILE'
+      ) {
+        // This is an edge case where the URL updates before the rest of the store,
+        // leading to a false assumption (and a thrown error) that a profile exists in
+        // the store. For this particular instance, make sure and transition back into
+        // listing the zip files.
```

Comment:
Can you please add a quick description of the STR, I find the the current text difficult to understand. Something like `This happens when pressing the browser's back button until displaying the zip file list`.